w3-access.md

```diff
@@ -243,19 +422,21 @@ Service will respond with delegations where audience is `did:mailto:web:mail:ali
       "can": "*"
     }
   ],
+  "prf": [],
   "exp": null,
-  "sig": "..."
+  "s": {
+    "/": {
+      "bytes": "7aEDQJb...dJQ4"
+    }
+  }
 }
 ```
 
-## Sharing access with a friend
-
-> Access protocol could be utilized to share access with a friend who may have never used web3.storage.
-
-Alice wants to share access to her space with her friend Bob. She does not know if Bob has ever heard of web3.storage, but she knows his email address `bob@gmail.com` allowing her to delegate capabilities to it:
+Program invokes [access delegate] capability with the account delegation so it can be received anywhere Alice logs in with her account.
```

Comment:
```suggestion
The program invokes an [access delegate] capability with the account delegation so it can be received anywhere that Alice logs in with her account.
```